packages/extension-chakra-storefront/jest.config.js

```diff
@@ -18,7 +17,16 @@ module.exports = {
             '<rootDir>/node_modules/@tanstack/react-query/build/lib/index.js',
         '^is-what$': '<rootDir>/node_modules/is-what/dist/cjs/index.cjs',
         '^copy-anything$': '<rootDir>/node_modules/copy-anything/dist/cjs/index.cjs',
-        "^@salesforce/cc-datacloud-typescript$": "<rootDir>/node_modules/@salesforce/cc-datacloud-typescript/dist/index.js",
+        '^@salesforce/cc-datacloud-typescript$':
+            '<rootDir>/node_modules/@salesforce/cc-datacloud-typescript/dist/index.js',
```

Comment:
The will fix the error when running `npm run test` in the `extension-chakra-storefront`
```
Cannot find module '@ark-ui/react/format' from 'node_modules/@chakra-ui/react/dist/cjs/components/format/index.cjs'

    Require stack:
      node_modules/@chakra-ui/react/dist/cjs/components/format/index.cjs
      node_modules/@chakra-ui/react/dist/cjs/index.cjs
      src/utils/test-utils.js
      src/components/quantity-picker/index.test.jsx

       8 | import {render} from '@testing-library/react'
       9 | import {BrowserRouter as Router} from 'react-router-dom'
    > 10 | import {ChakraProvider} from '@chakra-ui/react'
         | ^
      11 | import PropTypes from 'prop-types'
      12 |
      13 | import theme from '../theme'
```

Reply:
I'm not 100% convinced that we should need to do this in order to have things work. Do you know why the import works in normal webpack runtime but not in jest?

I would create a ticket with the chakra team and or ensure we create a ticket to remove this in the future.

Reply:
There is an issue that is reported on Chakra issues. https://github.com/chakra-ui/chakra-ui/issues/9284. The author suggested that we should set up ESM, which I don't think is feasible for us because it would affect the rest of the codebase.

I think we can
- Keep current fix
- Create a ticket to remove this set up once we transition to ESM for our jest set up.